cmd/cmd.go

```diff
@@ -146,13 +146,28 @@ func titledHelp(cmd *cobra.Command) {
 }
 
 // printFlags INFO logs all the given flags in alphabetical order.
-func printFlags(ctx context.Context, flags *pflag.FlagSet) {
+func printFlags(ctx context.Context, flags *pflag.FlagSet) error {
 	ctx = log.WithTopic(ctx, "cmd")
 
+	logLevel := flags.Lookup("log-level")
+	logFmt := flags.Lookup("log-format")
+
+	if logLevel != nil && logFmt != nil {
```

Comment:
best not do two things at once, call this manually before `printFlags`

Reply:
sure